src/Event/ValueObjects/LocationId.php

```diff
@@ -4,14 +4,13 @@
 
 namespace CultuurNet\UDB3\Event\ValueObjects;
 
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
 use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\IsString;
 
 final class LocationId
 {
     use IsString;
 
-    public const NIL_LOCATION = '00000000-0000-0000-0000-000000000000';
```

Comment:
Apart from the systemUser I cannot find any uses for a Nil Uuid apart from Location. Since we do not have Nil-events, organizers, Media,... This kinda feels too permissive to me. But I'm open for discussion.